src/pages/hardware/protocolReference/ble.tsx

```diff
@@ -0,0 +1,1042 @@
+            changes will generate a new log.</p>
+        <h2>Generic Timer</h2>
+        <p>The Generic Timer allows keeping track of an elapsed time. The Units Of Time are not specifically defined and can be chosen by the implementor. It may make sense to measure some durations in
+            hours, while others may warrant seconds. The Units Of Time may continuously increment. A new log will not be saved until a new event counter value is reported. The Generic Timer can be
+            associated with other data types. For example, you can associate Generic Timer 1 with temperature to indicate the time when a chosen temperature threshold was exceeded.</p>
+        <h2>Wakeup Event</h2>
+        <p>A custom parameter is used to configure the IOX-BT to wake up periodically to check for any wakeup events from beacons within range. The wakeup duration is 1s every 30s while sleeping. This
+            periodic wakeup can be enabled using the following custom parameter:</p>
+        <code className="small-code-sample">{`<Parameter Description='Enable Periodic Bluetooth Wakeup' Offset='167' Bytes='80' IsEnabled='true'/>`}</code>
+        <p>The implementor of this protocol should increase the frequency of advertisements sent during an attempted wakeup event. We recommend a 100ms advertisement interval that persists for a minimum of
+            1 minute.</p>
+        <p>When sending the wakeup event as part of the advertisement data, a value of 0x00 means “no event”. Anything greater than 0 that has not already been reported will cause the GO device to wake up
+            and report on the beacon advertisements. The event is only used as an indication for reporting any changes in the rest of the advertisement data. The actual contents of the alert event byte will
+            not be sent/reported.</p>
+        <h2>Custom Data</h2>
+        <p>Arbitrary data can be placed in the custom data segment. The data will not be interpreted by MyGeotab, but will be accessible through the API. The onus is on the implementor to extract and
+            interpret the data. The data must be preceded by the length. The length is limited by the amount of data that can fit in the optional information section. The maximum custom data length is 18
+            bytes. Any data changes will generate a new log.</p>
+        <div className="table-container">
+            <table>
+                <thead>
+                    <tr>
+                        <th>Offset</th>
+                        <th>Description</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td>1</td>
+                        <td>Length</td>
+                    </tr>
+                    <tr>
+                        <td>2 to Length</td>
+                        <td>Custom Data</td>
+                    </tr>
+                </tbody>
+            </table>
+            <p>Example:</p>
+        </div>
```

Comment:
```suggestion
        </div>
        <p>Example:</p>
```
`<p>` tag needs to be outside of the `<div>`